homeassistant/components/sensor/recorder.py

```diff
@@ -33,16 +36,21 @@
     DEVICE_CLASS_BATTERY: {"mean", "min", "max"},
     DEVICE_CLASS_ENERGY: {"sum"},
     DEVICE_CLASS_HUMIDITY: {"mean", "min", "max"},
+    DEVICE_CLASS_MONETARY: {"sum"},
```

Comment:
This little one seems to have sneaked in :)


```suggestion
```

Reply:
I sorted the constants alphabetically; https://github.com/home-assistant/core/pull/52299/files#diff-9c1e2390bb903b5318308ae555633181b80142865fc4e997ee24b3570860acc9L38

Reply:
No, just sorted alphabetically :) Moved from line 38

Reply:
I'm grabbing another coffee ☕ 
And maybe breakfast or something 🤦